spec/intl.html

```diff
@@ -1415,7 +1415,7 @@ <h1>
           </h1>
           <dl class="header">
             <dt>description</dt>
-            <dd>It performs implementation-defined processing to find the year of the current era for the date corresponding to _datetime_ in the context of the calendar represented by the string _calendar_ and returns it. The value should be an integer for calendars that have eras and *undefined* otherwise.</dd>
+            <dd>It performs implementation-defined processing to find the era for the date corresponding to _date_ in the context of the calendar represented by _calendar_ and returns an integer representing the ordinal position of the year of _date_ in that era, or *undefined* for calendars that do not have eras.</dd>
```

Comment:
Does "ordinal position" imply being 0-based or 1-based?  A few calendars are zero based while the vast majority are 1-based. But both exist. Should that possible variation be mentioned here?

Reply:
I had the same question, actually. Are there any concrete examples for which `eraYear` should be zero in an existing CLDR calendar?

Reply:
AFAIK, only one: `'buddhist'`.

```
new Date(-544, 11, 26).toLocaleDateString('en-US', {calendar: 'buddhist'})
// => '1/1/0 BE'
```

This is taken into account in the polyfill. 

https://github.com/tc39/proposal-temporal/blob/66f7464aaec64d3cd21fb2ec37f6502743b9a730/polyfill/lib/calendar.mjs#L1618-L1624

`calendarIsVulnerableToJulianBug` refers to https://bugs.chromium.org/p/chromium/issues/detail?id=1173158. 

The output of `toLocaleDateString` is actually wrong (at least I think it's wrong!) because it's using Julian instead of proleptic Gregorian for early dates.

The polyfill will throw for until that Chromium issue is addressed.

```js
Temporal.PlainDate.from('-000543-01-01[u-ca=buddhist]').eraYear
// RangeError: calendar 'buddhist' is broken for ISO dates before 1582-10-15 (see https://bugs.chromium.org/p/chromium/issues/detail?id=1173158)
```

Reply:
Thanks, that's great. I added a note covering both 0-indexed and reverse-advancing era years, although I used the Burmese calendar for the former example because it has multiple eras and is an official calendar of Myanmar (at least [according to Wikipedia](https://en.wikipedia.org/wiki/Burmese_calendar#Epoch)).